backend/src/main/java/com/google/capstone/EditImageInfo.java

```diff
@@ -0,0 +1,42 @@
+package com.google.capstone;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * Updates chosen property of the image in Datastore.
+ */
+@WebServlet("/api/v1/edit-info")
+public class EditImageInfo extends HttpServlet {
+
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String param = request.getParameter("selection");
```

Comment:
Maybe we can add a sanity check here to see if `param` is a valid one. So that we won't have random property data in our database.